service-commands/scripts/whitenoise.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Generates a random whitenoise mp3 file of given input size.
+ *
+ * Script usage: node whitenoise.js <size> <outFile>
+ */
+
+const { spawn } = require('child_process')
+const ffmpeg = require('ffmpeg-static')
+
+async function run() {
+  try {
+    const { size, outFile } = parseArgs()
+    whitenoise(size, outFile)
+  } catch (err) {
+    console.error(err.message)
+  }
+}
+
+/**
+ * Process command line args, defaults to 1 KiB file size, and unix timestamp for filename
+ */
+function parseArgs() {
+  const args = process.argv.slice(2)
+  const size = args[0] || 1
+  const outFile = args[1] || `whitenoise-${new Date().getTime()}.mp3`
+
+  // check appropriate CLI usage
+  if (!size || !outFile) {
+    const errorMessage = `Incorrect script usage for input size (${size}) and outFile (${outFile}).\nPlease follow the structure 'node whitenoise.js <size> <outFile>'`
+    throw new Error(errorMessage)
+  }
+
+  return { size, outFile }
+}
+
+/**
+ * Generate mp3 file using ffmpeg from given filesize in KiB
+ */
+function whitenoise(size, outFile) {
+  return new Promise((resolve, reject) => {
+    const process = spawn(ffmpeg, [
```

Comment:
would be great to add comments for what this is doing, both overall and each arg
for something like this, i also find it useful to link to the docs or stackoverflow post etc

Reply:
I second this! could we also add a comment where the output file is stored?

Reply:
Thanks, resolved 👍

Reply:
the link to the ffmpeg docs is 👍 ! could we still add a comment per flag about what it represents? 
e.g. 
```
`anoisesrc=d=${(1024 * size) / 8064}` /* generate a noise audio signal for the duration of ____ */
```

Reply:
:) do you think we can also add comments for the other flags in this spawn method too?

Reply:
just want to make it as easy as possible for the team to understand what's going on without having to refer to the docs

Reply:
yep, we should add this - easy to add and will save lots of repeated lookups in future

Reply:
Done, thanks!